### ISSUE TITLE
get verified by TransferWise again

### ISSUE THREAD

**Issue by whit537** [gratipay/inside.gratipay.com#381]
We made our first TransferWise payment in #249. Now downstream of [FD3028](https://gratipay.freshdesk.com/helpdesk/tickets/3028) they're giving us trouble again about verification and specifically about our name change(s). It seems that we're making some headway, though. The latest:

> Hi Chad,
Thank you for your email.
I have checked over all email correspondence and documents that were uploaded previously I did find on page of he documents that you sent in an amendment for [].
I will send this information to the verification team and have them relook it over.
You will be updated with an email once it’s done.
Thank you for your patience.
Kindest regards,

Ticket: https://gratipay.freshdesk.com/helpdesk/tickets/3271

Blocking:

 - [ ] https://gratipay.freshdesk.com/helpdesk/tickets/3028
 - [ ] https://gratipay.freshdesk.com/helpdesk/tickets/3284

**Comment by whit537** [comment]
Here's the beginning of the conversation:

> Hi Chad,
I hope you are doing great.
I can see you have set up a transfer on our website which is great.
However due to recent changes in regulations we would need to ask some additional information from you to fully verify Gratipay, LLC
Can you please provide us with one of the following:
- Certified copy of Articles of Organization
- Certified copy of Certificate of Formation
- Filing Receipt
>
You can simply reply to this email and attach a copy of one of these documents.
Looking forward to hear from you,

-----------------

> Thanks, []. What constitutes a "certified" copy?

--------------

>Hi Chad,
Sorry for not being clear in my previous email.
By certified we mean that it has to be signed and/or stamped or filed by the Secretary of State or other officially appointed legal entity by the State.
I hope this clears thing up and we will be waiting for your reply.
Best regards,

--------------------

>Thanks, []. I already uploaded these documents on July 3, 2015. Here is our internal ticket about it:
>
https://github.com/gratipay/inside.gratipay.com/issues/249
>
Why do you need these documents again?

----------------------

>Hi Chad,
Thanks for your reply!
We recently switched partner banks to better serve our US customers, and acquired some new regulations in the process.
I’ve taken a look at your ticket, and do see the document in question. However, the Certificate of Organization you provided is showing a company name of Zeta Web Design Company, not Gratipay.
You can either upload a new document, or simply attach it in a reply to this email, and we’ll get your account updated. :)
I hope this finds you well! Thanks for your cooperation during this exciting transition.
All the best,

-----------------

>Thanks for the explanation, []. Please recheck the tickets on your side. As documented in [our ticket](https://github.com/gratipay/inside.gratipay.com/issues/249), we provided both a) the documentation for each of our name changes, and b) our CP 575 showing our EIN for Gratipay, LLC. Can you find those documents on your side?

**Comment by whit537** [comment]
> Hi Chad,
Hope you are doing well today.
It’s a quick e-mail to let you know that we have finished business verification. Everything is fine now and we have resumed your payment. Money will be debited from your bank account later today. As soon as it clears on our bank account we’ll notify you with automated e-mail as usual and start conversion.
Once again thank you very much for your help on cooperation.
If there is anything else we can help you with don’t hesitate to let us know. We’ll be happy to help.
Best regards,

--------------------

> Thanks, []! I've got two more payments waiting in the wings, so now that we're unblocked I will get those started! :-)

Actually, I think it's just one.
